<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menedżer haseł</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplikacja zablokowana</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="create_pass_db">
       <property name="text">
        <string>Utwórz bazę haseł</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_pass_db">
       <property name="text">
        <string>Otwórz bazę haseł</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_app">
       <property name="text">
        <string>Zamknij</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Nowy</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
